### ISSUE TITLE
TIGER download hangs on unzip error

### ISSUE THREAD

**Issue by kochis** [pelias/interpolation#232]
When getting some unzip errors during the download processes, the process just hangs indefinitely (there's not more output once the error occurs):
```
error: [interpolation(TIGER)] Failed to unzip tl_2016_31081_addrfeat.zip
error: [interpolation(TIGER)]  message=invalid signature: 0x6d783f3c, stack=Error: invalid signature: 0x6d783f3c
    at /code/pelias/interpolation/node_modules/unzip/lib/parse.js:59:13
    at runCallback (timers.js:705:18)
    at tryOnImmediate (timers.js:676:5)
    at processImmediate (timers.js:658:5)
```

It seems like the `finish` event might not be firing in the case of an unzip error: https://github.com/pelias/interpolation/blob/master/script/js/update_tiger.js#L125

The stream might need to manually be closed in that case.

**Comment by missinglink** [comment]
Possibly related https://github.com/mafintosh/pump/blob/master/README.md#what-problem-does-it-solve

**Comment by orangejulius** [comment]
We just merged #235 to switch which library we use for handling ZIP files. The new one probably works better, so let us know if this is still an issue.

**Comment by missinglink** [comment]
also https://github.com/pelias/interpolation/pull/239 will likely solve this issue, I'm going to close this Craig, let us know if it rears its ugly head again ;)
